02-express-tutorial/answers/02-http-app.js

```diff
@@ -0,0 +1,53 @@
+const http = require("http");
+const { readFileSync } = require("fs");
+
+//get all files
+const homePage = readFileSync("./navbar-app/index.html");
+const homeStyles = readFileSync("./navbar-app/styles.css");
+const homeImage = readFileSync("./navbar-app/logo.svg");
+const homeLogic = readFileSync("./navbar-app/browser-app.js");
```

Comment:
This is a relative path to file, so when I execute this file I get an error because the path is incorrect. It should be `../navbar-app/${file}`.